package.json

```diff
@@ -31,7 +31,10 @@
     "@angular/forms": "^2.2.0",
     "@angular/http": "^2.2.0",
     "@angular/platform-browser": "^2.2.0",
+    "@types/es6-promise": "0.0.32",
```

Comment:
I got the `Cannot find name 'Promise'` error. I changed to use firebase.Promise after import 'firebase'